community/maturity.md

```diff
@@ -0,0 +1,99 @@
+
+More details can be found [here](https://community.apache.org/apache-way/apache-project-maturity-model.html).
+
+## Status of this assessment
+
+This assessment is evaluated during Uniffle's graduation, which is finished on 2024-11-29.
+
+## Maturity model assessment
+
+The following table is filled according to the [Apache Maturity Model](https://community.apache.org/apache-way/apache-project-maturity-model.html). Mentors and community members are welcome to comment and modify it.
+
+### CODE
+
+| **ID**   | **Description**                                                                                                                                                                                                                                                | **Status**                                                                                                                                                                                                          |
+| -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **CD10** | The project produces Open Source software for distribution to the public, at no charge.                                                                                                                                                                        | **YES** The project source code is licensed under the `Apache License 2.0`.                                                                                                                                         |
+| **CD20** | Anyone can easily discover and access the project's code..                                                                                                                                                                                                     | **YES** The [official website](https://uniffle.apache.org/) includes `GitHub` link which can access the project's repository on GitHub directly.                                                                    |
+| **CD30** | Anyone using standard, widely-available tools, can build the code in a reproducible way.                                                                                                                                                                       | **YES**  Apache OpenDAL provide `how-to-build` document for every component to tell user how to compile on bare metal, such as the [core's](https://github.com/apache/opendal/blob/main/core/CONTRIBUTING.md).      |
```

Comment:
Seems there's an extra space between `**YES**` and `Apache OpenDAL`? 
And why is it `Apache OpenDAL` here? There are about 10 instances where it's written as `OpenDAL` in this file. Is it a typo?

Reply:
Correct. Removed OpenDAL.